dom.go

```diff
@@ -110,12 +110,29 @@ type HTML struct {
 	properties, attributes          map[string]interface{}
 	eventListeners                  []*EventListener
 	children                        []ComponentOrHTML
+	new                             bool
 }
 
 // Node returns the underlying JavaScript Element or TextNode.
 func (h *HTML) Node() *js.Object { return h.node.(wrappedObject).j }
 
-func (h *HTML) restoreText(prev *HTML) {
+func (h *HTML) newNode() {
```

Comment:
Actually, given every DOM function here is `create[Element|ElementNS|TextNode]` how about `createNode` ?